packages/cx-layout/src/components/Header/token.ts

```diff
@@ -40,16 +67,9 @@ const Base = asHeaderToken({
     }),
   },
   Layout: {
-    // @todo move to spacing
-    Wrapper: 'lg:pt-15px',
-    Container: as(
-      cxElement.withContainerLayout,
-      'flex items-center justify-between lg:items-start flex-wrap',
-      // @todo move this to spacing
-      'md:h-auto h-12',
-    ),
-    ButtonsWrapper: 'order-1',
-    MenuContainer: 'lg:mt-9px absolute lg:static lg:w-full lg:order-1',
+    Container: 'container mx-auto flex justify-between items-center py-4',
+    MenuContainer: 'hidden lg:flex justify-between items-center flex-grow',
+    UtilityMenu: 'flex items-center',
```

Comment:
Shouldn[t this one be hidden on mobile too?

Reply:
You mean the UtilityMenu? Its element now lives inside MenuContainer, so it's already hidden by it.

Reply:
ok great.